ofproto/ofproto-dpif-xlate.c

```diff
@@ -8527,18 +8528,20 @@ xlate_add_static_mac_entry(const struct ofproto_dpif *ofproto,
     }
 
 #if defined(P4OVS)
-    /* Static MAC is configured, program P4 forwarding table */
-    struct xport *ovs_port = get_ofp_port(xbundle->xbridge,
-                                          in_port);
-    struct mac_learning_info fdb_info;
-    memset(&fdb_info, 0, sizeof(fdb_info));
-
-    if (!get_fdb_data(ovs_port, dl_src, &fdb_info)) {
-        ConfigFdbTableEntry(fdb_info, true);
-        ofproto->ml->p4_bridge_id = ovs_port->xbundle->p4_bridge_id;
-    } else {
-        VLOG_DBG("Error retrieving FDB information, skipping programming "
-                 "P4 entry");
+    if (ENABLE_OVS_P4_OFFLOAD) {
+        /* Static MAC is configured, program P4 forwarding table */
+        struct xport *ovs_port = get_ofp_port(xbundle->xbridge,
```

Comment:
Check for null return value?

Reply:
if ovs_port  is NULL, this API get_fdb_data will return -1 and that will lead into else case, and we LOG a message